src/components/Body/Body.component.jsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { Switch, Route } from 'react-router-dom';
+
+import Layout from '../Layout';
+import HomePage from '../../pages/Home';
+import NotFound from '../../pages/NotFound';
+
+function Body() {
+  return (
+    <Layout>
+      <Switch>
```

Comment:
Should put `Switch` outside of `Layout` in case we have different layouts for different routes, this is for later scaling

Reply:
should it be a `Layout` tag before showing the component of each `Route`?

> `<Switch>`
> `  <Route>`
> `    <Layout>`
> `      <Component />`